Add Apache Doris and Wget File (#309)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/doris.md

---
name: Apache Doris
category: Database
description: Apache Doris is a high-performance, distributed SQL database designed for real-time data analytics.
download_url: https://github.com/apache/doris/releases
works_on_arm: true
supported_minimum_version: 
    version_number: 1.2.5
    release_date: 2023/06/18

optional_info:
    homepage_url: https://doris.apache.org/
    support_caveats:
    alternative_options: 
    getting_started_resources:
        arm_content: 
        partner_content: 
        official_docs: https://doris.apache.org/docs/gettingStarted/quick-start
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum: https://github.com/apache/doris/releases/tag/1.2.5-rc01
    release_notes__recommended_minimum: 

>     other_info: Binary is  present for Linux/ARM64. Doris version 1.2.5 has the ARM64 support.

Author: ```suggestion
    other_info: Binary for Doris Linux/ARM64 is released from version 1.2.5.
```
